crates/debugger_tools/TODO.md

```diff
@@ -0,0 +1 @@
+- [ ] housekeeping: ensure references to debug adapters are cleaned when they are closed
```

Comment:
I think it would be better if we moved the TODO list to one of the zed_debugger sub channels to be more in line with the rest of the code base